src/App.js

```diff
@@ -1,34 +1,15 @@
 import logo from "./logo.svg";
 import "./App.css";
-const todoList = [
-  {
-    id: 1,
-    title: "My First Assignment in CODE THE Dream.",
-  },
-  {
-    id: 2,
-    title: "I will be try to complete Assigment.",
-  },
-  {
-    id: 3,
-    title: "Finally I am completed my assignment.",
-  },
-  {
-    id: 4,
-    title: "LOVE YOU ALL CODE THE DREAM.",
-  },
-];
+import TodoList from "./components/TodoList";
+import AddTodoForm from "./components/AddTodoForm";
+import { Fragment } from "react";
+
 function App() {
   return (
-    <>
-      <h1>My First Assigment on CODE THE DREAM</h1>
-      {/** Render the items */}
-      <ul>
-        {todoList.map((item) => {
-          return <li key={item.id}>{item.title}</li>;
-        })}
-      </ul>
-    </>
+    <Fragment>
+      <TodoList />
+      <AddTodoForm />
```

Comment:
Good use of self-closing components. Using the "/" is an XML convention that allows you to combine the opening and closing tags when you don't need both.